tabs/adjustments.js

```diff
@@ -15,7 +15,19 @@ TABS.adjustments.initialize = function (callback) {
     }
 
     function get_rc_data() {
-        MSP.send_message(MSP_codes.MSP_RC, false, false, load_html);
+        MSP.send_message(MSP_codes.MSP_RC, false, false, init_adjustment_template);
+    }
+    
+    function init_adjustment_template() {
```

Comment:
What is your motivation here to put this into a separate function? (I am not saying that this is wrong, in general 'one function does one thing' is a good approach.)

But I do not think that function chaining is the right approach here, since there is no callback involved. `init_adjustment_template` should be a function of its own right, that is then called at the very beginning of `load_html`.

What I have started doing, when doing this kind of refactoring, is to, instead of adding one more function inside the closure, properly add the functions as properties of the object (`TABS.adjustments.initAdjustmentTemplate = function...`, see the end of `pid_tuning.js` for examples), in order to move to a better structured code base.